GitUI/CommandsDialogs/BrowseDialog/FormGitLog.cs

```diff
@@ -119,6 +123,30 @@ private void alwaysOnTopCheckBox_CheckedChanged(object sender, EventArgs e)
             alwaysOnTopCheckBox.Checked = TopMost;
         }
 
+        private void enhancedLogCheckBox_CheckedChanged(object sender, EventArgs e)
+        {
+            AppSettings.EnhancedGitLog = !AppSettings.EnhancedGitLog;
+            enhancedLogCheckBox.Checked = AppSettings.EnhancedGitLog;
+        }
+
+        private void SaveToFileToolStripMenuItem_Click(object sender, System.EventArgs e)
+        {
+            using (var fileDialog = new SaveFileDialog
+            {
+                Title = Name,
+                DefaultExt = ".txt",
+                AddExtension = true
+            })
+            {
+                fileDialog.Filter =
+                    "All files (*.*)|*.*";
+                if (fileDialog.ShowDialog(this) == DialogResult.OK)
+                {
+                    File.WriteAllText(fileDialog.FileName, AppSettings.GitLog.ToString());
```

Comment:
Log to file could request an enhanced format, for instance tab separated output, time in separate column